site2/docs/txn-use.md

```diff
@@ -20,11 +20,17 @@ This section provides an example of how to use the transaction API to send and r
 
 2. Enable transaction. 
 
-    Change the configuration in the `broker.conf` file.
+    Change the configuration in the `broker.conf` or `standalone.conf` file.
 
     ```
+    //mandatory configuration, used to enable transaction coordinator
     transactionCoordinatorEnabled=true
+   
+    //mandtory configuration, used to create systemTopic used for transaction buffer snapshot
     systemTopicEnabled=true
+   
+    //It is a mandatory configuration, if need to guanrantee exactly-once semantics.
```

Comment:
```suggestion
    //mandatory configuration, used to guarantee exactly-once semantics.
```

I think it is a must without conditions.

Reply:
If the user does not need to turn on deduplication, then this does not need to be turned on. Otherwise it's just a waste of performance

Reply:
Thanks for clarifying this. Then there will only be a typo fix.

Reply:
Oh my fault! Thanks for pointing out.